src/main/java/org/fundacion/jala/converter/models/Asset.java

```diff
@@ -55,88 +56,99 @@ public Asset(final String newTitle, final String newPath, final String newChecks
     }
 
     /**
-     * Obtains Id of the project
-     * @return id of the project
+     * Obtains Id of the audio or video converter.
```

Comment:
Maybe it should be more general, the id is for the assets not only audio or video files.

Reply:
ok